ci/sandbox/build-pipeline.yaml

```diff
@@ -600,8 +599,8 @@ spec:
         config:
           platform: linux
           params:
-            PROXY_NODE_URL: "https://test-proxy-node.((cluster.domain))"
-            STUB_CONNECTOR_URL: "https://test-connector.((cluster.domain))"
+            PROXY_NODE_URL: "https://eidas-proxy-node-test.((cluster.domain))"
+            STUB_CONNECTOR_URL: "https://eidas-stub-connector-test.((cluster.domain))"
```

Comment:
I'd revert these changes - this can cause issues in sandbox. We don't need to change the sandbox URLs.